src/main/java/pages/MenuPage.java

```diff
@@ -0,0 +1,38 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class MenuPage extends BasePage{
```

Comment:
Side menu isn't a separate page because it's available on any page after login. That's why we've added HomePage which should contain all common elements and methods like this menu

Reply:
The same for Basket button in the right top corner
<img width="451" alt="image" src="https://user-images.githubusercontent.com/14195681/178569443-c1a0878c-fe17-43fc-b69e-b1fc631e36bc.png">